algorithms/kernel/decision_tree/decision_tree_classification_predict_dense_default_batch_impl.i

```diff
@@ -53,12 +53,11 @@ using namespace decision_tree::internal;
 
 template<typename algorithmFPType, CpuType cpu>
 services::Status DecisionTreePredictKernel<algorithmFPType, defaultDense, cpu>::
-    compute(const NumericTable * x, const classifier::Model * m, NumericTable * y, const daal::algorithms::Parameter * par)
+    compute(const NumericTable * x, const classifier::Model * m, NumericTable * y, NumericTable * p, const daal::algorithms::Parameter * par)
 {
     typedef daal::services::internal::SignBit<algorithmFPType, cpu> SignBitType;
 
     DAAL_ASSERT(x);
-    DAAL_ASSERT(y);
```

Comment:
Why it was deleted?

Reply:
because now it can be null, if e.g. a user set only probabilities to compute